### DISCUSSION TITLE
Echostar HDS-600RS - Anyone tried with this

### DISCUSSION THREAD

**Discussion by fridayshoes** [GerryDazoo/Slinger#45] [Q&A]
Anyone know if this can be made to work with the Echostar HDS-600RS?

**Comment by GerryDazoo** [comment]
Looks like it might work but I don't have access to one to give it a try.

**Reply by fridayshoes** [reply]
What configuration differences are there between the different slingboxes?

**Reply by GerryDazoo** [reply]
Besides telling my code what type the slingbox is 500/350 Solo/Pro
They are all essentially the same. Some don't support HD, some don't have HDMI.

**Reply by GerryDazoo** [reply]
Did you give this a try? I'm interested in knowing if it worked.

**Reply by fridayshoes** [reply]
Not yet, I don't have a lot of time at the moment to figure out the setup for all of this, been keeping an eye out for a quick setup tutorial then I'll give it a go

**Comment by BobC1954** [comment]
I can confirm, works fine with Echostar HDS-600rs

**Reply by fridayshoes** [reply]
@BobC1954 Cool, what settings did you use?

**Reply by BobC1954** [reply]
sbtype="Solo/Pro/ProHD"
Password (it's admin password)
IP address (it's ip address)
Port (it's port)

Thats all.
